pkg/features/kube_features.go

```diff
@@ -299,7 +299,7 @@ const (
 	BalanceAttachedNodeVolumes featuregate.Feature = "BalanceAttachedNodeVolumes"
 
 	// owner: @kevtaylor
-	// beta: v1.15
+	// ga: v1.17
```

Comment:
Add GA as a new line so we can easily see feature graduation history

Reply:
Added graduation steps (I put the alpha one back in for completeness)